CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Please ensure your pull request adheres to the following guidelines:
 
 * New lessons or improvements to existing lessons are welcome.
 * Please check your spelling and grammar.
-* Please adhere to our [style guide](https://github.com/elixirschool/elixirschool/wiki/Lesson-Styleguide)
+* Please adhere to our [style guide](https://github.com/haskellschool/haskellschool/wiki/Lesson-Styleguide)
```

Comment:
should be `haskellfoundation/HaskellSchool/` right?

Reply:
yep, good catch